packages/core/src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -16,32 +17,32 @@ const { Provider } = radioButtonGroupContext;
 
 const RadioButtonGroup: React.FC<RadioButtonGroupProps> = ({
   direction = Direction.Vertical,
-  value,
-  onValueChange = () => {},
-  defaultValue,
+  value = "",
+  onValueChange,
+  defaultValue = "",
   style,
   children,
   ...rest
 }) => {
-  const [internalValue, setInternalValue] = React.useState<string | undefined>(
-    value || defaultValue
-  );
+  const [internalValue, setInternalValue] = React.useState<string>("");
 
   React.useEffect(() => {
-    if (value != null) {
-      setInternalValue(value);
-    }
+    const realValue = getValueForRadioButton(value);
```

Comment:
any reason you got rid of that `if (value != null)` here?

Reply:
@peter Same thing since the real value would have thrown if it is undefined or null